dfdatetime/posix_time.py

```diff
@@ -145,7 +247,8 @@ def _GetNormalizedTimestamp(self):
           determined.
     """
     if self._normalized_timestamp is None:
-      if self._timestamp is not None:
+      if (self._timestamp is not None and self._timestamp >= self._INT64_MIN and
```

Comment:
As above, please remove these bounds checks, and move them to something more specific if there's an implementation that limits these to 64 bit int